### PR TITLE
build: update Github organization name & use newer NPM dependencies

### PR DESCRIPTION
**Pull request by ringods**
Signed-off-by: Ringo De Smet <ringo@de-smet.name>

### CODE REVIEW

File: CITATION.cff
Comment (davidsneighbour) [inline CITATION.cff]: I would call them "Hugo Components / Component Name" to take the whole "vanity" part out of it. The citation file is for citing setup and the title doesn't need to be overly long. If you enable edits by maintainers then I'll change that before merging your PRs in all other projects.

**Review by davidsneighbour — APPROVED** [review]

File: CITATION.cff
Comment (ringods) [reply CITATION.cff]: @davidsneighbour I think I fixed them all. 😉

**Pull request merged**
